collection-scripts/common.sh

```diff
@@ -12,6 +12,7 @@ export SOS_DECOMPRESS=${SOS_DECOMPRESS:-1}
 export BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
 export SOS_PATH="${BASE_COLLECTION_PATH}/sos-reports"
 export SOS_PATH_NODES="${SOS_PATH}/_all_nodes"
+export METALLB_NAMESPACE=${METALLB_NAMESPACE:-"metallb-system"}
```

Comment:
@rabi I'm curious about one thing: does this namespace changes from upstream to downstream or between releases?

Reply:
it would be great to have a more verbose commitmsg or at least a description. I see the problem you're trying to fix, but probably a better description might help to get more context about the issue that you faced.

Reply:
approving since we need this fix anyway I guess

Reply:
I don't think the namespace changes, but the code was already there since https://github.com/openstack-k8s-operators/openstack-must-gather/pull/1/files#diff-6e4bacccc4d0b833b012d4abda2523635bcbdf747eb37c13433d14273fae3928R12

As for commit message, I don't think there anything more to add in description:) we need to collect resources in metallb namespace which has been missing.